metrics/ewma.go

```diff
@@ -36,78 +39,70 @@ func NewEWMA15() EWMA {
 	return NewEWMA(1 - math.Exp(-5.0/60.0/15))
 }
 
-// EWMASnapshot is a read-only copy of another EWMA.
-type EWMASnapshot float64
+// eWMASnapshot is a read-only copy of another EWMA.
+type eWMASnapshot float64
```

Comment:
```suggestion
type ewmaSnapshot float64
```
You may consider this form instead. It seems less common to break an acronym up by case, e.g. see `HTTP` - it doesn't often appear `hTTP`.